src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Semantic/Services/RazorSemanticTokensInfoService.cs

```diff
@@ -230,7 +230,7 @@ private static bool ContainsOnlySpacesOrTabs(SourceText razorSource, int startIn
         var end = startIndex + count;
         for (var i = startIndex; i < end; i++)
         {
-            if (razorSource[i] is not ' ' or '\t')
+            if (razorSource[i] is not (' ' or '\t'))
```

Comment:
This is subtle, nice catch!

Reply:
Would have been nicer to catch it when I wrote the code in the first place. Not the first time I've made this mistake either 🤦🏼‍♂️